### PR TITLE
Add documentation on how to use external blobstores

### PR DESCRIPTION
**Pull request by modulo11**
## WHAT is this change about?
Add documentation on how to use external blobstores.

## Does this PR introduce a change to `config/values.yml`?
No

## Acceptance Steps
Try it out.

## Tag your pair, your PM, and/or team
@phil9909 
@loewenstein

### CODE REVIEW

File: docs/platform_operators/external-blobstore.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    blobstore:
+    endpoint: <your-s3-compatible-blobstore-endpoint>
+    region: <your-region>
+    access_key_id: <your-access-key-id>
+    secret_access_key: <your-secret-access-key>
+    package_directory_key: <bucket-name-package>
+    droplet_directory_key: <bucket-name-droplet>
+    resource_directory_key: <bucket-name-resource>
+    buildpack_directory_key: <bucket-name-buildpack>
+    aws_signature_version: "4"
```
Comment (davewalter) [inline docs/platform_operators/external-blobstore.md:31]: Are these lines meant to be indented within the `blobstore` key?

File: docs/platform_operators/external-blobstore.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    ```
+
+    For AWS S3 the endpoint is `https://s3.<your-region>.amazonaws.com/`.
```
Comment (davewalter) [inline docs/platform_operators/external-blobstore.md:34]: Would it make sense to simply use this value as the endpoint in the YAML above?

File: docs/platform_operators/external-blobstore.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    ```
+    REGION=<your-region>
+    aws s3api create-bucket --bucket <bucket-name-buildpack> --region "$REGION" --create-bucket-configuration LocationConstraint="$REGION" --acl private
+    aws s3api create-bucket --bucket <bucket-name-droplet>   --region "$REGION" --create-bucket-configuration LocationConstraint="$REGION" --acl private
+    aws s3api create-bucket --bucket <bucket-name-package>   --region "$REGION" --create-bucket-configuration LocationConstraint="$REGION" --acl private
+    aws s3api create-bucket --bucket <bucket-name-resource>  --region "$REGION" --create-bucket-configuration LocationConstraint="$REGION" --acl private
+    ```
+
+    *Note*: The name of your buckets (`package_directory_key`, `droplet_directory_key`, `resource_directory_key`, `buildpack_directory_key`) need to be unique per region! Also check your needed/supported `aws_signature_version` and configure it accordingly.
```
Comment (davewalter) [inline docs/platform_operators/external-blobstore.md:13]: We think that the names of the buckets have to be globally unique, not just per region.

**Comment by loewenstein** [comment]
@davewalter We've got a 7 days old LGTM. What's blocking the merge?

**Pull request merged**